### PR TITLE
add new circle check

### PR DESCRIPTION
**Pull request by jeff-shin**

### CODE REVIEW

**Comment by SeanPollock** [comment]
screen shot 2018-03-13 at 1 51 42 pm

File: README.md [hunk 1]
```diff
@@ -17,3 +17,12 @@ This plugin will automatically apply styling to all inputs of `type="checkbox"`
 If you wish to disable the styling on a specific input, you must add `data-fancy-disabled="true"` attribute.
 
 You can also disable styling on all child inputs by adding `data-fancy-disabled="true"` to any ancestor element.
+
+# Generating Dist
+
+1. Run `npm install` to update/install dependencies
+2. Run `npm prepublishOnly` to generate 1dist.
```
Comment (6) [inline README.md]: minor - typo with `1dist`

File: dist/fancy-checkbox.js [hunk 2]
```diff
@@ -516,9 +516,14 @@
     $inputs.each(function() {
       var $input = $(this);
       if ($input.parents('[data-fancy-disabled="true"]').length === 0) {
+        radioClass = 'radio-fancy';
+        if ($input.hasClass('pretty')) {
+          radioClass += ' pretty';
+          console.log("WE made it!");
```
Comment (6) [inline dist/fancy-checkbox.js]: i would remove this :D

File: README.md [hunk 3]
```diff
@@ -17,3 +17,12 @@ This plugin will automatically apply styling to all inputs of `type="checkbox"`
...
 You can also disable styling on all child inputs by adding `data-fancy-disabled="true"` to any ancestor element.
+
+# Generating Dist
+
+1. Run `npm install` to update/install dependencies
+2. Run `npm prepublishOnly` to generate 1dist. 
+
+# Making a new release
+1. Bump the version in `package-lock.json`
+2. Run `npm publish` tp publish new version
```
Comment (SethDavenport) [inline README.md]: tp -> to

**Pull request merged**